tests/assertions/is_not_substring_of.zsh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env zunit
+
+@test 'not a substring of' {
+    assert 'foo' is_not_substring_of 'yellow'
```

Comment:
Can you pass the assertion to `run`, and then assert that the `$state` is zero and `$output` is empty.

See the other tests for examples: https://github.com/molovo/zunit/blob/master/tests/assertions/same_as.zunit